central/cluster/datastore/datastore.go

```diff
@@ -8,6 +8,8 @@ import (
 	"github.com/stackrox/rox/central/cluster/index"
 	clusterStore "github.com/stackrox/rox/central/cluster/store/cluster"
 	clusterHealthStore "github.com/stackrox/rox/central/cluster/store/clusterhealth"
+	complianceManager "github.com/stackrox/rox/central/complianceoperator/v2/compliancemanager"
```

Comment:
With these changes you need to remove the import of `complianceManager` in the cluster datastore.